### ISSUE TITLE
Documenting steps to clean up Landrush hosts entries

### ISSUE THREAD

**Issue by LalatenduMohanty** [projectatomic/adb-atomic-developer-bundle#525]
Sometimes because of old entries in Landrush `hosts.json` file  OpenShift console i.e.  https://openshift.cdk:8443/console or  https://openshift.adb:8443/console is not accessible. 
Fix:
- Do a `vagrant destroy`
-  Delete extra entries from 
  - (for Windows) C:/Users/<USER>/.vagrant.d/data/landrush/hosts.json
  - (for Linux and OS X) ~/.vagrant.d/data/landrush/hosts.json
    i.e. the `hosts.json` file should only contain below parenthesis

```
{
}
```

**Comment by LalatenduMohanty** [comment]
For a healthy Landrush setup you should be able to see similar output post the `vagrant up`

```
$ vagrant.exe landrush ls
cdk.vm                         172.28.128.3
3.128.28.172.in-addr.arpa      cdk.vm
cdk                            cdk.vm
```

**Comment by praveenkumar** [comment]
@LalatenduMohanty landrush provide option to remove domain using cli so we don't have to manually delete the entry.

```
$ vagrant landrush help
vagrant landrush <command>

commands:
  {start|stop|restart|status}
    control the landrush server daemon
  list, ls
    list all DNS entries known to landrush
  dependentvms, vms
    list vms currently dependent on the landrush server
  set { <host> <ip> | <alias> <host> }
    adds the given host-to-ip or alias-to-hostname mapping.
    existing host ip addresses will be overwritten
  rm, del { <host> | <alias> }
    delete the given hostname or alias from the server
  help
    you're lookin at it!
```

**Comment by jeffmaury** [comment]
Make sure landrush files were cleaned,but this got errors when performing vagrant up (using http://cdk-builds.usersys.redhat.com/builds/nightly/29-Aug-2016/):

```
C:\Users\JeffMAURY\CDK\29082016\components\rhel\rhel-ose>vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'cdkv2.2'...
==> default: Matching MAC address for NAT networking...
==> default: Setting the name of the VM: rhel-ose_default_1472504216455_53472
==> default: Fixed port collision for 22 => 2222. Now on port 2200.
==> default: Clearing any previously set network interfaces...
==> default: [landrush] virtualbox requires an additional private network; adding it
==> default: [landrush] starting dns server
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2200 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2200
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Registering box with vagrant-registration...
    default: Would you like to register the system now (default: yes)? [y|n]y
    default: username: jmaury@redhat.com
    default: password:
==> default: Checking for guest additions in VM...
    default: No guest additions were detected on the base box for this VM! Guest
    default: additions are required for forwarded ports, shared folders, host only
    default: networking, and more. If SSH fails on this machine, please install
    default: the guest additions and repackage the box to continue.
    default:
    default: This is not an error message; everything may continue to work properly,
    default: in which case you may ignore this message.
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Landrush IP not installed in guest yet (or it's an outdated version). Installing now.
[landrush] Using eth1 (172.28.128.5)
==> default: [landrush] adding machine entry: cdk.vm => 172.28.128.5
==> default: [landrush] adding static DNS entry: cdk => cdk.vm
[landrush] Using eth1 (172.28.128.5)
[landrush] starting 'Wired AutoConfig' service
L’erreur système 5 s’est produite.

Accès refusé.

[landrush] Unable to start 'Wired AutoConfig' service. Unable to configure DNS on host. Try manual configuration.
==> default: The folowing SH comand responded with a non-zero exit status.
==> default: Vagrant asumes that this means the comand failed!
==> default: sudo rm /etc/docker/ca.pem & sudo systemctl restart docker
==> default: Stdout from the comand:
==> default: Stder from the comand:
==> default: rm: canot remove ���/etc/docker/ca.pem���: No such file or directory

C:\Users\JeffMAURY\CDK\29082016\components\rhel\rhel-ose>ping cdk
La requête Ping n’a pas pu trouver l’hôte cdk. Vérifiez le nom et essayez à nouveau.

C:\Users\JeffMAURY\CDK\29082016\components\rhel\rhel-ose>vagrant landrush ls
cdk.vm                         172.28.128.5
5.128.28.172.in-addr.arpa      cdk.vm
cdk                            cdk.vm
```

**Comment by LalatenduMohanty** [comment]
@optak I could not reproduce this on WIndows7. Can you retry and see if you can reproduce the issue?

**Comment by Preeticp** [comment]
Closed via #559

**Comment by Preeticp** [comment]
This is documented the troubleshooting guide at https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/troubleshooting.adoc
